guix-systole/packages/itk.scm

```diff
@@ -1,111 +1,132 @@
 (define-module (guix-systole packages itk)
-    #:use-module (gnu packages image-processing)
-    #:use-module (guix build-system cmake)
-    #:use-module (guix download)
-    #:use-module (guix packages)
-    )
+  #:use-module (gnu packages image-processing)
+  #:use-module (guix build-system cmake)
+  #:use-module (guix build-system trivial)
+  #:use-module (guix download)
+  #:use-module (guix git-download)
+  #:use-module ((guix licenses)
+                #:prefix license:)
+  #:use-module (guix packages))
 
 (define-public itk-slicer
   (package
     (inherit insight-toolkit)
     (version "5.4.0")
```

Comment:
We should override the name of the package: `(name "itk-slicer)` to keep consistency with the vtk naming